src/components/Search/SelectedRevisionsTable/SelectedRevisionsTable.tsx

```diff
@@ -24,14 +24,13 @@ const tableHeaderDetails: TableHeaderDetails[] = [
   { id: 5, title: 'Timestamp', alignment: 'left' },
 ];
 
-function SelectedRevisionsTable(props: SelectedRevisionsProps) {
+function SelectedRevisionsTable(
+  props: SelectedRevisionsProps,
+): JSX.Element | null {
   const { revisions } = props;
   const dispatch = useDispatch();
 
-  if (revisions.length === 0) {
-    return <div className="missingRevisions">Add some revisions</div>;
-  }
-  return (
+  return revisions.length > 0 ? (
```

Comment:
Instead of doing the conditional rendering here, it should be done in SearchView, so that we aren't loading this component at all unless there are selected revisions revisions.
See an example here: https://github.com/mozilla/perfcompare/blob/master/src/components/Search/SearchView/SearchView.tsx#L61-L63

Reply:
right now do we have something to keep track of selected revisions?

Reply:
If you look at the example provided above, it's doing the same thing. If there are no search results (and input is not focused, that bit isn't relevant here), then the results list does not render or mount at all.

You'd be doing basically the same thing in SearchView using mapStateToProps